h3ronpy/tests/arrow/test_vector.py

```diff
@@ -20,3 +21,25 @@ def test_geometry_to_cells_central_park():
     arr = geometry_to_cells(point, 8).to_numpy()
     assert len(arr) == 1
     assert arr[0] == h3.geo_to_h3(point.y, point.x, 8)
+
+
+def test_coordinate_values_are_not_equal_issue_58():
+    # Step 1: Create a point (latitude and longitude)
+    lat, lon = 37.7749, -122.4194  # Example coordinates (San Francisco)
+    point = Point(lon, lat)  # shapely expects (longitude, latitude)
+
+    # Step 2: Convert the point to an H3 cell (resolution 9 for example)
+    resolution = 9
+    h3_cells = geometry_to_cells(point, resolution)
+
+    # Step 3: Convert the H3 cell back to WKB points
+    wkb_points = cells_to_wkb_points(h3_cells)
+
+    assert len(wkb_points) == 1
+
+    # Step 4: Decode the WKB point to a Shapely geometry
+    for wkb_point in wkb_points:
```

Comment:
I know this is just a test file and speed doesn't matter as much (and maybe this is a list of one?) but `shapely.from_wkb` is vectorized and much faster than a loop and `shapely.wkb.loads`.

Reply:
I did not know that. Here I was lazy and just took the code from the original issue and copied it into a unittest ;)